pages/catalog/index.php

```diff
@@ -20,6 +20,7 @@
     case 'category':
     case 'fullSize':
     case 'thumbnail':
-        define('HANDLER_CLASS', 'PKP\pages\catalog\PKPCatalogHandler');
+        define('HANDLER_CLASS', 'CatalogHandler');
+        import('pages.catalog.CatalogHandler');
```

Comment:
So I have in index.php

```
switch ($op) {
    case 'category':
    case 'fullSize':
    case 'thumbnail':
        define('HANDLER_CLASS', 'APP\pages\catalog\CatalogHandler');
        break;
}
```
and in pages/catalog/CatalogHandler.inc.php I have

```
namespace APP\pages\catalog;

use APP\handler\Handler;

class CatalogHandler extends Handler
{
```

But I get 
`PHP Fatal error:  Uncaught TypeError: get_class_methods(): Argument #1 ($object_or_class) must be an object or a valid class name, string given in /ojs-master/lib/pkp/classes/core/PKPPageRouter.php:283`

Not sure what the problem is, probably something with the namespace, but not sure what

Reply:
I am not sure if this solves that problem, but however your new classes should be end with .php instead of .inc.php -- this is what I only see for now :-)

Reply:
Oh god, how did I miss that 😂